deneme3.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+from py2neo import authenticate, Graph,Node
+
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--user', type=str, default="neo4j", help='username for neo4j')
+parser.add_argument('--password', type=str, default="777", help='password of the user')
+
+args = parser.parse_args()
+
+authenticate("localhost:7474", args.user, args.password)
+graph = Graph()
+start=graph.begin()
+#g=Graph("bolt://localhost:7687", auth=("neo4j", "777"))
+
+#string="CREATE (Node:Actor{userid: "", fullname: ""})"
+
+path=r"C:\Users\ASUS\Documents\staj\data\FILMS.txt"

```

Comment:
pathleri relative directory olarak alalım ki kod'un yeri veya datanın yeri değişse de hata almamış olalım.
Örneğin: ..\data\FILMS.txt
Gene bu tür constant bilgileri argument parsing ile command linedan, kullanıcıdan, alsak çok daha güzel olur. Default değerine relative konum yazabiliriz